react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { HTML: false, CSS: false, JS: false };
+    this.toggleHTML = this.toggleHTML.bind(this);
+    this.toggleCSS = this.toggleCSS.bind(this);
+    this.toggleJS = this.toggleJS.bind(this);
+  }
+
+  toggleHTML() {
+    this.setState({ HTML: true, CSS: false, JS: false });
+  }
+
+  toggleCSS() {
+    this.setState({ HTML: false, CSS: true, JS: false });
+  }
+
+  toggleJS() {
+    this.setState({ HTML: false, CSS: false, JS: true });
+  }
```

Comment:
The accordion shouldn't know that it's showing HTML, CSS, and JavaScript. Please see the constraint section of the assignment.